redash/destinations/hipchat.py

```diff
@@ -13,6 +13,7 @@
 }
 
 
+@deprecated()
```

Comment:
@arikfr not sure you actually wanted to deprecate this (and the URL data source) as part of this PR, so let me know if you want me to revert this commit.

Reply:
No, it's cool. We do need to update the docs about the URL data source though.

Reply:
The page currently mixes `JSON` and `URL` and feels like it might need a slight rewrite (or some good chunks related to `URL` removed). WDYT @susodapop?